src/application/authorizers/conseiller-inter-structure-milo-authorizer.ts

```diff
@@ -54,7 +54,6 @@ export class ConseillerInterStructureMiloAuthorizer {
 
         if (
           isSuccess(jeuneMilo) &&
-          jeuneMilo.data.idStructureMilo &&
```

Comment:
si les 2 sont undefined on laisse passer ?

Reply:
`conseillerMilo.data.structure.id` ne peut pas être `undefined`

Reply:
you sure about this ? ça me parait bizarre
le conseiller albert durant en prod il n'a pas de structure milo par exemple

Reply:
Dans `ConseillerMiloSqlRepository.get` :point_down: 
```ts
if (
  !conseillerSqlModel.idStructureMilo ||
  !conseillerSqlModel.structureMilo
) {
  return failure(new ConseillerMiloSansStructure(id))
}
```